Updated design of main window UI

SecretImbecile opened the pull request:
Simple design changes to UI widgets within the main window (including all nested widgets) to make them look more consistent and less cramped. The minimum window size for the application has been increased to **1280x720**.

Before:
![Screenshot of main window before update](https://github.com/user-attachments/assets/b5bf2cc1-c723-49a2-ac3b-609bc398695c)
After:
![Screenshot of main window after update](https://github.com/user-attachments/assets/d57d9b99-17d1-472a-b9e9-665df991fa9c)

This PR includes more changes than the last, and a number of UI elements have been renamed/re-referenced in the code. I would appreciate it if you could test the application (add/modify some track sections) to ensure this has been done correctly.

## Feedback Requested

If you have any opinions on how things look, please comment on this pull request _before_ merging, I'll commit any requested changes.

SecretImbecile committed:
Updated layout of main window

Improved splitter behaviour. Some dynamic widgets have had their parents changed in the code.

SecretImbecile committed:
Updated nested widgets in main window

SecretImbecile committed:
Corrections to Project Widget

- Remove frame decoration around track section buttons
- Fix spacing between STL inputs
- Correct typo on button 'Browse' -> 'Browse...'
- Fix horizontal overflow by reducing minimum widget size

SecretImbecile commented:
Addressed feedback in commit https://github.com/H27CK/openFVD/pull/2/commits/4bf7f84a7d00122a74b7b2980d2929972691a694

- Remove frame decoration around track section buttons
- Fix spacing between STL inputs
- Correct typo on button 'Browse' → 'Browse...'
- Fix horizontal overflow by reducing minimum widget size

H27CK commented:
`QLineEdit` for `stlEdit1` and `texEdit` in `projectWidget` need to be aligned with left border.

SecretImbecile committed:
Correct grid layout in project widget

An extra column was present in the grid layout, adding some horizontal spacing that wasn't intended.

SecretImbecile commented:
> `QLineEdit` for `stlEdit1` and `texEdit` in `projectWidget` need to be aligned with left border.

That's corrected:

![image](https://github.com/user-attachments/assets/57830827-546a-4cdb-b2f5-9afe3424c570)
